packages/db/src/Component/ScriptDialogueLine.tsx

```diff
@@ -96,17 +97,54 @@ const DialogueBasic = (props: {
     }
 };
 
+
+const DialogueGenderPopover = (props: {
+    female: DialogueBasicComponent[],
+    male: DialogueBasicComponent[]
+}) => {
+    const { female, male } = props;
+
+    const femaleComponent = female.map((component, index) => (
+        <DialogueBasic component={component} />
+    ))
+
+    const maleComponent = male.map((component, index) => (
+        <DialogueBasic component={component} />
+    ))
+
+    const maleToolTip = (props: any) => (
+        <Tooltip {...props}>
+          {femaleComponent}
+        </Tooltip>
+    );
+
+    return (
+        <OverlayTrigger
+            placement="top"
+            delay={{ show: 250, hide: 400 }}
+            overlay={maleToolTip}
+        >
+            <span style={{textDecoration: "underline"}}>
+                {maleComponent}
+            </span>
+        </OverlayTrigger>
+    )
+}
+
+
 const DialogueChild = (props: {
     component: DialogueChildComponent;
     index?: number;
 }) => {
     const { component, index } = props;
     switch (component.type) {
         case ScriptComponentType.DIALOGUE_GENDER:
-            const femaleComponents = component.female.map((component) => (
-                <DialogueBasic component={component} />
-            ));
-            return <>{mergeElements(femaleComponents, "")}</>;
+
+            const femaleComponents = component.female;
+            const maleComponents = component.male;
+            const popover = <DialogueGenderPopover female={femaleComponents} male={maleComponents} />
+
+            return <>{mergeElements([popover], "")}</>;
```

Comment:
You can just return `<DialogueGenderPopover female={component.female} male={component.male} />` here